Tchap/Modules/Room/Members/RoomParticipantsViewController.m

```diff
@@ -26,9 +26,11 @@
 
 #import "MXCallManager.h"
 
+#import "ContactTableViewCell.h"
+
 #import "RageShakeManager.h"
 
-@interface RoomParticipantsViewController () <UITableViewDelegate, UITableViewDataSource, UISearchBarDelegate, UIGestureRecognizerDelegate, MXKRoomMemberDetailsViewControllerDelegate, ContactsViewControllerDelegate, RoomParticipantsInviteCoordinatorBridgePresenterDelegate>
+@interface RoomParticipantsViewController () <UITableViewDelegate, UITableViewDataSource, UISearchBarDelegate, UIGestureRecognizerDelegate, MXKRoomMemberDetailsViewControllerDelegate, RoomParticipantsInviteCoordinatorBridgePresenterDelegate>
```

Comment:
Why do we still use a specific Tchap `RoomParticipantsViewController` class? (I think we can switch to the Element one)

Reply:
This is removed in next PR (code cleaning).